src/components/CollectionDetailPage/CollectionDetailPage.tsx

```diff
@@ -32,13 +30,33 @@ import CollectionItem from './CollectionItem'
 
 import './CollectionDetailPage.css'
 
-const STORAGE_KEY = 'dcl-collection-notice'
-
 export default class CollectionDetailPage extends React.PureComponent<Props, State> {
   state: State = {
     tab: this.props.tab || ItemType.WEARABLE
   }
 
+  componentDidMount(): void {
+    const { collection } = this.props
+    if (collection) {
+      this.fetchCollectionForumPostReply()
+    }
+  }
+
+  componentDidUpdate(prevProps: Props): void {
+    const { collection } = this.props
+    if (!prevProps.collection && collection) {
```

Comment:
Should we add `collection && !collection.forumPostReply` to avoid re-fetches?

Reply:
This condition will be fulfilled only when the user enters the `CollectionDetails` directly without previously fetching all the collections in the `CollectionsPage` and the `CollectionProvider` updates the state.

Also, we want to fetch the forum post replies every time the user enters the `CollectionDetails` because we don't know when a curator left a new comment in the forum post.

Reply:
makes sense! good catch